src/media/js/views/feed/feed_shelf.js

```diff
@@ -10,14 +11,11 @@ define('views/feed/feed_shelf', ['jquery', 'l10n', 'utils', 'utils_local', 'z'],
 
         var slug = args[0];
         builder.start('feed/feed_shelf.html', {
-            landing: true,
             slug: slug
```

Comment:
could, but most builder.starts look like this so it's easy to spot. plus we always might add more vars